### PR TITLE
Added publisher and size shown in win uninstall

### PR DESCRIPTION
**Pull request by demetris-manikas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,8 @@ config.json.sample:
   "win" : {
     "title" : "Loopline Systems",
     "version" : "x.x.x.x",
+    "publisher": "Publisher Info",
+    "size": "Estimated Size",
```
Comment (stefanjudis) [inline README.md]: Can we add an example value here -> `200MB`?
The sample file shows mostly concrete values over the place. :)

**Comment by stefanjudis** [comment]
Just a tiny note - over all looking good. :+1:

File: README.md (hunk 1 above)
Comment (demetris-manikas) [inline README.md]: Replaced Estimated Size with a number. The units are KB as stated later on in the file.

**Comment by demetris-manikas** [comment]
Any thoughts about adding
BrandingText "${APP_NAME} ${APP_VERSION}" to the template?
This way NullSoft Install System vX.XX.XXX from the (un)installer

**Comment by stefanjudis** [comment]
I'll do `electron-builder` night tonight. :) 

Sorry for the huge delay. :(

**Comment by stefanjudis** [comment]
> Any thoughts about adding BrandingText "${APP_NAME} ${APP_VERSION}" to the template?

Sounds perfect to me. :)

About the size - would be cool if we could automatically add the size, as we have the application available. 

What do you think?

**Comment by demetris-manikas** [comment]
I found a method to add the size but I 'll have no Windows PC available for a while so I cannot test it.

It is described in http://nsis.sourceforge.net/Add_uninstall_information_to_Add/Remove_Programs Computing Estimated Size section.

Take a look. Looks quite straightforward to implement.

And... while you are there read the whole page. 
There are quite a few uninstall options that are not implemented
The QuietUninstallString is very easy to implement along with some others (VersionMajor, VersionMinor).
Let me now what you think.

**Comment by demetris-manikas** [comment]
I managed to get hold of windows and tried it. Looks fine.

**Comment by stefanjudis** [comment]
@demetris-manikas can you please resolve conflicts... After that I'll give it a try.

Thanks a lot. :)

**Comment by demetris-manikas** [comment]
Conflicts resolved.

**Comment by demetris-manikas** [comment]
Hi, any news on this one? 
I guess you are busy but I would appreciate some feedback.
Thanks.

**Comment by stefanjudis** [comment]
@demetris-manikas sorry it happened again. :(

There was an older PR than yours which I merged first. Sorry about that.

**Comment by demetris-manikas** [comment]
I guess you I have to resolve conflicts once more... not a problem for this one.

**Comment by demetris-manikas** [comment]
Conflicts resolved.

**Comment by stefanjudis** [comment]
:+1: Tried it and works.

**Pull request merged**